### PR TITLE
Finished off Advanced Medical Refresh

### PR DESCRIPTION
**Pull request by MrFlufficans**
Hopefully I didn't make it too wordy, Looking forward to heavy scrutiny

### CODE REVIEW

**Comment by grayherm** [comment]
You spelled personnel wrong on line 139 other than that id approve it :)

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md [hunk 1]
```diff
@@ -2,71 +2,162 @@
...
+Self Interacting with '**Ctrl + Windows key**' will allow you to treat yourself.  
+Interacting on another person using '**Windows key**' will let you treat them.  
+
+When interacting with yourself you will be presented with a list of all your body parts.  
+For patients you will need to interact with that specific body part by looking at the body part in question and interacting.  
+\
+\
+There are visual and/or audio cues that will let you know if you have medical issues.  
+
+Pulsing white border around your screen indicates you are in pain.
```
Comment (zuedev) [inline docs/3-advanced-techniques/4-advanced-medical-procedures.md:20]: We use [painEffectType = 2](https://github.com/uagpmc/missions/blob/master/userconfig/cba_settings.sqf#L214), which is [chromatic aberration](https://github.com/acemod/ACE3/blob/master/addons/medical_feedback/initSettings.sqf#L7).

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md [hunk 2]
```diff
@@ -2,71 +2,162 @@
...
+When interacting with yourself you will be presented with a list of all your body parts.  
+For patients you will need to interact with that specific body part by looking at the body part in question and interacting.  
+\
+\
+There are visual and/or audio cues that will let you know if you have medical issues.  
+
+Pulsing white border around your screen indicates you are in pain.  
+Losing saturation from your vision (Turning Black and white) indicates blood loss.  
+Blood spots on the edge of your screen indicates open wounds.  
+Loud ringing can indicate hearing damage or tinnitus.
```
Comment (zuedev) [inline docs/3-advanced-techniques/4-advanced-medical-procedures.md:23]:
```suggestion
Loud ringing can indicate hearing damage or tinnitus.  This can be prevented in most scenarios by using ear plugs.
```

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md [hunk 3]
```diff
@@ -2,71 +2,162 @@
...
 ## 3. Common Injuries
 
-Out of the many injuries that you may encounter, a common one will be velocity, which occurs when you get shot or damaged by shrapnel. Other wounds include Avulsions, Contusions, Abrasions and more.
+You will encounter many different types of injuries.  
+Such as Velocity, Avulsions, Contusions, Abrasions, Lacerations, Cuts, Punctures and Crush.  
+Not only are there different types but also Intensities, Minor, Medium and Large.  
+
+These mostly tell you the cause of an injury, In most cases velocity and avulsions are caused from gunshot wounds.  
+You can also suffer from Fractures these are depicted by a Red bone and are treated using a Splint.  
+There is a chart further below that will tell you what bandages work best for certain wounds.
```
Comment (zuedev) [inline docs/3-advanced-techniques/4-advanced-medical-procedures.md:33]:
```suggestion
There is a [chart](#_4a-bandages) further below that will tell you what bandages work best for certain wounds.  
```

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md [hunk 4]
```diff
@@ -2,71 +2,162 @@
...
+**Field Dressings** - Run of the mill Bandage can be used on most wounds aside from Velocity and Abrasions.  
+**Packing Bandages** - Same effectiveness as a Field dressing with higher reopening chance.  
+**Elastic Bandages** - The Butter to your Bread and Butter, Very high effectiveness but also very high reopening chance.  
+**QuikClot** - The Bread to your Bread and Butter, Mediocre effectiveness with very low chance of reopening.  
+
+Below is a Chart that will reflect the specifics.  
 ![Effectiveness of Bandages](https://ace3mod.com/img/wiki/feature/medical1.jpg)
 
-Elastics are very efficient in closing wounds, but only temporarily as they will almost certainly re-open. QuickClot on the other hand is very efficient in keeping wounds closed
+Using Quikclots first followed by Elastics will be your Bread and Butter for treating Wounds.
```
Comment (zuedev) [inline docs/3-advanced-techniques/4-advanced-medical-procedures.md:52]:
```suggestion
Using Quikclots first followed by Elastics will be your Bread and Butter for treating Wounds. It is recommended that riflemen should take 10 of each.
```

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md [hunk 5]
```diff
@@ -2,71 +2,162 @@
...
-The Surgical Kit is carried and used by the medic, and it is used to stitch up wounds that reopen. One surgical kit has multiple uses
-The PAK is for fully healing someone, restoring their heart rate and blood pressure. The PAK is one use, and fairly heavy, meaning it should only be used when the patient is either heavily wounded to the point that using a PAK will waste fewer resources that fully bandaging the patient, or when they are down, with no heart rate, etc. The speed of PAKing someone varies depending on their state; if they’re heavily wounded it will take longer.
+There are three kits available to medics, The **PAK (Personal Aid Kit)**, The **Surgical Kit** and lastly the **Defibrillator**.  
+
+The **PAK** or **Personal Aid Kit** when used on a person after a lengthy time will heal them fully, this item is a one time use.  
+The best time to use this item is when you have no other choice, example being either you will run out of medical to heal them or if you are on the clock and getting them up is mandatory.  
+Note: PAK's are very heavy and will take a long time to use based on how wounded the Patient and will vanish after use.
```
Comment (zuedev) [inline docs/3-advanced-techniques/4-advanced-medical-procedures.md:60]: I'm pretty sure our PAKs don't work like this. You can only use a PAK when the patient is both fully stabilised and has enough blood in them. It is used for removing wounds whereas surgical kits (which can be used when the patient is just only stable) are just for closing them.

I think we should test this somewhat soon on the server itself to figure out which approach is correct. 😅

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md [hunk 6]
```diff
@@ -2,71 +2,162 @@
...
+
+The **PAK** or **Personal Aid Kit** when used on a person after a lengthy time will heal them fully, this item is a one time use.  
+The best time to use this item is when you have no other choice, example being either you will run out of medical to heal them or if you are on the clock and getting them up is mandatory.  
+Note: PAK's are very heavy and will take a long time to use based on how wounded the Patient and will vanish after use.  
+
+The **Surgical Kit** on the other hand is reusable and is used to stitch wounds after bandaging.  
+The Surgical kit should only be used if you have adequate protection or if not using it will result in the death of a squadmate or yourself.  
+Depending on the amount of wounds needing to be stitched the time required will differ with more wounds taking a longer time to complete.  
+
+The **Defribrillator** does what you would expect, It restores a heartrate to the patient.
```
Comment (zuedev) [inline docs/3-advanced-techniques/4-advanced-medical-procedures.md:66]: Should we emphasise the patient can be killed if wounds are not bandaged/stitched prior to defibbing? I vaguely remember that as being an issue in the past.

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md [hunk 7]
```diff
@@ -2,71 +2,162 @@
...
-Blood bags – pretty obvious, they restore blood in a patient. You can only use a blood bag on someone else, so if you’re low, just ask anyone to give you a transfusion. There are 250, 500 and 1000ml variants.
-Morphine will suppress pain, but will also decrease the heart rate of the patient
-Epinephrine will increase the heart rate
+**Blood**
+As the name would imply these are blood bags which come in 3 different sizes, 250, 500 and lastly 1000.  
+These are best used in a safe position or under protection from fellow operators,  
+It takes time to apply the Transfusion and is the only way to restore blood to the patient aside from a PAK.  
+
+**Plasma and Saline**
+As of yet, Plasma and Saline serve the same function as Blood and serve no additional benefits.
```
Comment (zuedev) [inline docs/3-advanced-techniques/4-advanced-medical-procedures.md:99]:
```suggestion
As of yet, Plasma and Saline serve the same function as Blood and serve no additional benefits. "Piss bags" also serve the same purpose.
```

**Review by zuedev — CHANGES_REQUESTED** [review]
Very well done! Just some small considerations from me.

**Review by HANZZZZZZZZ — DISMISSED** [review]

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md (hunk 5 above)
Comment (MrFlufficans) [reply docs/3-advanced-techniques/4-advanced-medical-procedures.md:60]: +1 on needing to test the specifics of PAK's I can't quite recall our specifics for it

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md (hunk 6 above)
Comment (MrFlufficans) [reply docs/3-advanced-techniques/4-advanced-medical-procedures.md:66]: I can't say I recall if it was since in almost all cases I have had the patient bandaged and stitched prior to defibbing, needs further testing

**Comment by Luigikissme** [comment]
the mans took out the memes im disappointed in you fluffy

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md (hunk 3 above)
Comment (MrFlufficans) [reply docs/3-advanced-techniques/4-advanced-medical-procedures.md:33]: I am unsure of what this does, Is it a hyperlink?

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md (hunk 1 above)
Comment (MrFlufficans) [reply docs/3-advanced-techniques/4-advanced-medical-procedures.md]: Changed Line to.
`Your vision being affected by chromatic aberration or distortion indicates you are in pain.`

**Comment by MrFlufficans** [comment]
Fuck you spell checking

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md (hunk 3 above)
Comment (zuedev) [reply docs/3-advanced-techniques/4-advanced-medical-procedures.md:33]: Yes! It's called a [heading anchor](https://vuepressbook.com/tutorial/tutorial2.html#creating-links).

**Comment by zuedev** [comment]
That spell checking action was a mistake. I've already removed it from upstream so it should be gone for good once this is merged.

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md (hunk 5 above)
Comment (zuedev) [reply docs/3-advanced-techniques/4-advanced-medical-procedures.md:60]: Made an issue for it #20, will assign after this is merged to keep this moving.

File: docs/3-advanced-techniques/4-advanced-medical-procedures.md (hunk 6 above)
Comment (zuedev) [reply docs/3-advanced-techniques/4-advanced-medical-procedures.md:66]: I'll run that by Jeremy first, lets get this one merged :>

**Review by zuedev — DISMISSED** [review]

**Comment by MrFlufficans** [comment]
Committed the Heading Anchor suggestion, should be all set to merge now

**Review by Luigikissme — APPROVED** [review]
Really disappointed with the removal of memes in this change but i will accept begrudgingly
